### PR TITLE
Externalizing user messages in account plugin

### PR DESCRIPTION
**Pull request by meltsufin**
Addresses Issue #397.

@patflynn PTAL

### CODE REVIEW

File: google-account-plugin/resources/messages/AccountBundle.properties [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#
+
+login.toolbar.button.sign.in.text="Sign in to Google..."
+
+login.panel.sign.in.button.text=Sign In
+login.panel.sign.out.button.text=Sign Out
+login.panel.add.account.button.text=Add Account
+login.panel.play.console.link.text=Open Google Developers Console
+login.panel.cloud.console.link.text=Open Play Developer Console
+login.panel.play.console.link.url=https://play.google.com/apps/publish/#ProfilePlace
```
Comment (patflynn) [inline google-account-plugin/resources/messages/AccountBundle.properties]: is that #ProfilePlace getting comment stripped?

File: google-account-plugin/resources/messages/AccountBundle.properties [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+login.panel.cloud.console.link.url=https://console.developers.google.com/
```
Comment (patflynn) [inline google-account-plugin/resources/messages/AccountBundle.properties]: generally I'm not crazy about having url links in message bundles. If we do eventually get translators, I don't want them to be able to change these.

Comment (meltsufin) [inline google-account-plugin/resources/messages/AccountBundle.properties]: I was thinking about the translators too and was wondering if they would potentially point to a different URLs. But if that's not a concern, I'll just remove those.

File: google-account-plugin/resources/messages/AccountBundle.properties (hunk 1 above)
Comment (meltsufin) [inline google-account-plugin/resources/messages/AccountBundle.properties]: doesn't look like it, but I'm going to undo the inclusion of URLs in the bundle

File: google-account-plugin/src/com/google/gct/login/ui/UsersListCellRenderer.java [hunk 3]
```diff
@@ -53,12 +54,15 @@
  * how each user item in the Google Login panel would be displayed.
  */
 public class UsersListCellRenderer extends JComponent implements ListCellRenderer {
-  private final static String CLOUD_LABEL_TEXT = "Open Google Developers Console";
-  private final static String PLAY_LABEL_TEXT = "Open Play Developer Console";
+  private final static String CLOUD_LABEL_TEXT = AccountMessageBundle.message(
```
Comment (patflynn) [inline google-account-plugin/src/com/google/gct/login/ui/UsersListCellRenderer.java]: break after AccountMessageBundle

File: google-account-plugin/src/com/google/gct/login/util/AccountMessageBundle.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+/**
+ * Bundle class to get account plugin messages from resources/messages
+ */
+public class AccountMessageBundle {
+  @NonNls
+  private static final String BUNDLE_NAME = "messages.AccountBundle";
+  private static Reference<ResourceBundle> bundleReference;
+
+  private static ResourceBundle getBundle() {
+    ResourceBundle bundle = com.intellij.reference.SoftReference.dereference(
```
Comment (patflynn) [inline google-account-plugin/src/com/google/gct/login/util/AccountMessageBundle.java]: same here. Break at the 'higher' syntactic level.

Comment (patflynn) [inline google-account-plugin/src/com/google/gct/login/util/AccountMessageBundle.java]: this should fit on one line

File: google-account-plugin/src/com/google/gct/login/ui/UsersListCellRenderer.java (hunk 3 above)
Comment (meltsufin) [inline google-account-plugin/src/com/google/gct/login/ui/UsersListCellRenderer.java]: I don't think there's a need for a break at all.

**Comment by patflynn** [comment]
LGTM. Squash and commit.

**Pull request merged**